pype/plugins/tvpaint/create/create_render_layer.py

```diff
@@ -0,0 +1,150 @@
+from avalon.tvpaint import pipeline, lib
+
+
+class CreateBeauty(pipeline.Creator):
```

Comment:
Can we please call this `createRenderlayer`? it should match the file name for simpler maintenance.